src/interceptor/injection-scripts.js

```
@@ -0,0 +1,252 @@
+/**
+ * Dynamic Injection Script Loader
+ * Fetches provider-specific injection scripts from the backend and executes them
+ * This script is injected into the main world of the website
+ */
+
+(function() {
+    'use strict';
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)